.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+#### What? Why?
+
+
+#### What should we test?
+
+
+#### How does this relate to the Spree upgrade?
```

Comment:
Maybe something a bit more specific like: "Any known conflicts with the Spree Upgrade?" would be easier to describe? Just a thought, happy either way.